public/js/utils.js

```diff
@@ -212,9 +213,10 @@ export function copyToClipboard(str) {
 
 export async function getBundlephobiaSize(name, version) {
   try {
+    const port = portStore.getStore();
     const {
       gzip, size, dependencySizes
-    } = await getJSON(`https://bundlephobia.com/api/size?package=${name}@${version}`);
+    } = await getJSON(`http://localhost:${port}/bundle/${name}/${version}`);
```

Comment:
It is not necessary to know the port here. Just request `/bundle/${name}/${version}` (fetch by default will use the localhost with the current port).

Reply:
Wasn't sure about that thanks a lot ^^ I'll clean it a bit.